.github/workflows/publish-quil-py.yml

```diff
@@ -81,10 +81,40 @@ jobs:
         name: wheels
         path: dist
 
+  windows: 
+    runs-on: windows-latest 
+    strategy: 
+      matrix: 
+        python-version: ['3.8', '3.9', '3.10', '3.11'] 
+        target: [x64, x86] 
+    steps: 
+      - uses: actions/checkout@v2 
+      - uses: dtolnay/rust-toolchain@nightly
+      - uses: actions/setup-python@v4 
+        with: 
+          python-version: ${{ matrix.python-version }} 
+          architecture: ${{ matrix.target }} 
+      - name: Build wheels 
+        uses: messense/maturin-action@v1 
+        env:
+          RUSTFLAGS: "-Z macro-backtrace"
+        with: 
+          target: ${{ matrix.target }} 
+          rust-toolchain: nightly
+          args: -i python --release --manifest-path quil-py/Cargo.toml --out dist
```

Comment:
Why are we using `nightly` here?

Reply:
I switched to nightly so I could apply the `-Z macro-backtrace` flag and see where the x86 build was failing. I meant to switch it back, but missed it. I'll test with the stable toolchain and restore that step to how it was.